core/src/main/java/org/elasticsearch/search/profile/query/CollectorResult.java

```diff
@@ -45,6 +45,7 @@
     public static final String REASON_SEARCH_MIN_SCORE = "search_min_score";
     public static final String REASON_SEARCH_MULTI = "search_multi";
     public static final String REASON_SEARCH_TIMEOUT = "search_timeout";
+    public static final String REASON_SEARCH_CANCELED = "search_canceled";
```

Comment:
should we be consistent with the number of `l`? there is only one here while you put 2 in `cancellation`? (feel free to ignore, it could easily a special case in english that I don't know about!)